internal/staticanalysis/obfuscation/collect_data.go

```diff
@@ -41,7 +41,14 @@ func CollectData(parserConfig js.ParserConfig, jsSourceFile string, jsSourceStri
 		return nil, nil
 	}
 
-	data := RawData{}
+	// don't want null values in JSON
+	data := RawData{
+		Identifiers:    make([]string, 0),
```

Comment:
nit: consider just literals for these which are shorter but equivalent.

e.g.

```golang
Identifiers: []string{}
```

Alternatively, could we just annotate the struct field with `omitempty` and have the JSON consumer handle non-existent fields?

Reply:
Happy to use the shorter literals but I'm not sure about `omitempty`, it feels like it creates more problems than it solves.
Additionally, there might end up being another useful meaning for a non-existent key or null value, so for the case of seeing none of a particular kind of data (when no errors occur) it makes sense to me to just use an empty array